src/Rules/Library/BoundingRectangleNotNull.cs

```diff
@@ -14,11 +14,11 @@ class BoundingRectangleNotNull : Rule
     {
         public BoundingRectangleNotNull()
         {
-            this.Info.Standard = A11yCriteriaId.ObjectInformation;
-            this.Info.PropertyID = Axe.Windows.Core.Types.PropertyType.UIA_BoundingRectanglePropertyId;
-            this.Info.Description = Descriptions.BoundingRectangleNotNull;
-            this.Info.HowToFix = HowToFix.BoundingRectangleNotNull;
-            this.Info.ErrorCode = EvaluationCode.Error;
+            Info.Standard = A11yCriteriaId.ObjectInformation;
```

Comment:
The intent is that you can read through each Rule and know exactly what it does, so the repetitive nature is part of that. If we refactor this, I'd probably lean toward a parameterized constructor for the `Rule` class, so that some of the properties could be `readonly`. @RobGallo might have some thoughts on this, as well